proto/minder/v1/minder.proto

```diff
@@ -1887,27 +1887,64 @@ message ListEvaluationResultsResponse {
 message RestType {
     message Fallback {
         int32 http_code = 1 [(buf.validate.field).int32 = { gte: 100, lte: 599 }];
-        string body = 2;
+        string body = 2 [
+            (buf.validate.field).string = {
+                pattern: "^\\s*\\{.*\\}\\s*$",
+                max_len: 1000,
+            },
+            (buf.validate.field).ignore = IGNORE_IF_DEFAULT_VALUE
+        ];
     }
 
     // endpoint is the endpoint to fetch data from.
     // This can be a URL or the path on the API.bool
     // This is a required field and must be set.
     // This is also evaluated via a template which allows
     // us dynamically fill in the values.
-    string endpoint = 1;
+    string endpoint = 1 [
+        (buf.validate.field).string = {
+            pattern: "^(https:\\/\\/[a-zA-Z0-9.-]+)?(\\/[a-zA-Z0-9._-]+)*(\\/\\{\\{[^}]+\\}\\})*(\\/)?$",
+            max_len: 400,
+        }
+    ];
 
     // method is the method to use to fetch data.
-    string method = 2;
+    string method = 2 [
+        (buf.validate.field).string = {
+            pattern: "^(GET|POST|PUT|PATCH|DELETE)$",
+        }
+    ];
 
     // headers are the headers to be sent to the endpoint.
-    repeated string headers = 3;
+    repeated string headers = 3 [
+        (buf.validate.field).repeated = {
+            items: {
+                string: {
+                    pattern: "^[a-zA-Z0-9-]+: .+$",
+                    max_len: 400,
+                }
+            }
+        },
+        (buf.validate.field).ignore = IGNORE_IF_DEFAULT_VALUE
+    ];
 
     // body is the body to be sent to the endpoint.
-    optional string body = 4;
+    optional string body = 4 [
+        (buf.validate.field).string = {
+            pattern: "^\\s*\\{.*\\}\\s*$",
```

Comment:
Again, like `fallback`, this may not be useful validation, and we may need to excuse it.